cmd/logger/config.go

```diff
@@ -51,6 +51,30 @@ const (
 	defaultTarget = "_"
 )
 
+// NewConfig - initialize new logger config.
+func NewConfig() Config {
+	cfg := Config{
+		// Console logging is on by default
+		Console: Console{
+			Enabled: true,
+		},
+		HTTP:  make(map[string]HTTP),
+		Audit: make(map[string]HTTP),
+	}
+
+	// Create an example HTTP logger
+	cfg.HTTP["_"] = HTTP{
```

Comment:
use `defaultTarget` instead of `_`
```suggestion
	cfg.HTTP[defaultTarget] = HTTP{
```